### PR TITLE
added phrg rubocop.yml and spec

### PR DESCRIPTION
**Pull request by connorgallopo**
Removed gemfile.lock from the gitignore and updated it. 

Also added the phrg rubocop.yml and related spec file. Fixed any new failures.

Noticed an issue when running `rspec`
```
Gem::LoadError:
  You have already activated rack 2.0.5, but your Gemfile requires rack 1.6.10. Prepending `bundle exec` to your command may solve this. 
```

I worked on troubleshooting but could not find a solid solution. If you run  `learn`  instead of `rspec` the issue does not occur.

### CODE REVIEW

File: .results.json [hunk 1]
```diff
@@ -0,0 +1 @@
...n\u001b[31m      \u001b[0m{\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mtitle\u001b[0m: \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mTalisman\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35martist_name\u001b[0m: \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mAir\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mrelease_year\u001b[0m: \u001b[1;34m2007\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mreleased\u001b[0m: \u001b[1;36mtrue\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mgenre\u001b[0m: \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mPost-Rock\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0m}\u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNameError:\u001b[0m\n\u001b[31m  uninitialized constant Song\u001b[0m\n\u001b[36m# ./spec/models/song_spec.rb:3:in `\u003ctop (required)\u003e'\u001b[0m\n"],"examples":[],"summary":{"duration":0.000473,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":2},"summary_line":"0 examples, 0 failures, 2 errors occurred outside of examples"}
```
Comment (garettarrowood) [inline .results.json]: Whats the story with this file?

**Comment by garettarrowood** [comment]
Does `rspec` error out? Or does it just throw a warning?

File: .results.json (hunk 1 above)
Comment (connorgallopo) [reply .results.json]: Ahh ill remove it. This is the file that generates when you run `learn`.

Comment (garettarrowood) [reply .results.json]: Oh! Ok. Thanks!

**Comment by connorgallopo** [comment]
It errors out @garettarrowood

**Review by garettarrowood — APPROVED** [review]
If `learn` is working, this is good to go :)!

**Pull request merged**